pipeline_steps/common.groovy

```diff
@@ -2336,7 +2336,7 @@ Boolean willOverlapMaintenanceWindow(){
 Boolean issueExistsForNextMaintenanceWindow(project="RE"){
   LocalDateTime nextWindow = getNextMaintenanceWindowStart()
   String nextWindowDateString = nextWindow.format(DateTimeFormatter.ISO_LOCAL_DATE)
-  List issues = jira_query("project=\"${project}\" AND summary ~ '${nextWindowDateString} Maintenance Window'")
+  List issues = jira_query("project=\"${project}\" AND summary ~ '${nextWindowDateString} Maintenance Window' and STATUS not in (Finished)")
```

Comment:
Note the same clause was used previously: https://github.com/rcbops/rpc-gating/blob/8e1bd5bb739b7fa2816ebf2f082dad9fe6cde3c0/pipeline_steps/common.groovy#L2382